Detectors/TRD/simulation/include/TRDSimulation/TRDSimParams.h

```diff
@@ -37,6 +37,10 @@ struct TRDSimParams : public o2::conf::ConfigurableParamHelper<TRDSimParams> {
   float maxMCStepSize = 0.1;                              ///< maximum size of MC steps
   bool doTR = true;                                       ///< switch for transition radiation
   SimParam::GasMixture gas = SimParam::GasMixture::Xenon; ///< the gas mixture in the TRD
+  // TRF parameters
+  int trf = 0;                  // Sampled TRF function. 0: default TRF, 1: TRF described in TRF TDR, 2: No TRF, 3: Landau dist as TRF (parameters specified below)
```

Comment:
Maybe could include also an enum for this? Like with the GasMixture:
```
enum class TRF: int {
 Sampled,
 TDR,
 Delta,
 Landau,
 Default = Sampled
};
```

I think this would make it more verbose than just using this integer but this probably personal taste :smile: